integration/src/test/resources/org/mule/streaming/payload-statistics-non-repeatable-config.xml

```diff
@@ -46,6 +55,18 @@
 
         <test-components:queue-push config-ref="listOfMessagesSourceComplete"/>
     </flow>
+    
+    <flow name="streamSource" initialState="stopped">
+        <marvel:magneto-mutant-summon>
```

Comment:
if these new sources do not support disabling repeatable streaming, move the tests to its own test case to avoid running them twice